packages/admin-stories/src/admin-rte/RteCustomInlineStyles.tsx

```diff
@@ -0,0 +1,46 @@
+import { IRteOptions, IRteRef, makeRteApi, Rte } from "@comet/admin-rte";
+import { Box, Card, CardContent } from "@material-ui/core";
+import { Highlight } from "@material-ui/icons";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+import { PrintEditorState, useAutoFocus } from "./helper";
+
+const rteOptions: IRteOptions = {
+    supports: ["header-one", "header-two", "bold", "italic", "strikethrough", "sub", "sup"],
+    customInlineStyles: {
+        HIGHLIGHT: {
+            label: "Highlight!",
+            icon: Highlight,
+            style: {
+                color: "blue",
+                textDecoration: "underline",
```

Comment:
it can be a bit confusing to use default link styling as highlight styling (I'd use backgroundColor: "yellow")

Reply:
You're right! I've changed it to `backgroundColor: "yellow"` only.